src/vapid-helper.js

```diff
@@ -164,7 +172,7 @@ function validateExpiration(expiration) {
  * @param  {string} publicKey       The VAPID public key.
  * @param  {string} privateKey      The VAPID private key.
  * @param  {string} contentEncoding The contentEncoding type.
- * @param  {integer} [expiration]   The expiration of the VAPID JWT.
+ * @param  {number} [expiration]   The expiration of the VAPID JWT.
```

Comment:
Please revert this change, as it is unrelated (you can submit it in a separate PR).